extensions/legit/peek-legit.lisp

```diff
@@ -68,8 +68,6 @@ Notes:
 (define-attribute highlight
   (t :background :base0D))
 
-(defvar *collector*)
```

Comment:
I have mixed feelings because now we need the `collector` variable in front of many function calls :/ Is there an obvious advantage?

IMO a global variable is OK in CL.